src/topology/compact_open.lean

```diff
@@ -103,6 +103,45 @@ continuous_iff_continuous_at.mpr $ assume ⟨f, x⟩ n hn,
 
 end ev
 
+section Inf_induced
+
+-- not a very conceptual proof!
```

Comment:
Can you add a docstring explaining the math-content of the lemma?

Reply:
```suggestion
/-- The compact-open topology on `C(α, β)` is equal to the infimum of the compact-open topologies
on `C(s, β)` for `s` a compact subset of `α`.  The key point of the proof is that the union of the
compact subsets of `α` is equal to the union of compact subsets of the compact subsets of `α`. -/
```